Template/Tag/EtrackerTag.web.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+        var etrackerConfig = parameters.get('etrackerConfig', {});
+        var trackingType = parameters.get('trackingType');
+        if (trackingType === 'pageview') {
+            var etrackerID = etrackerConfig.etrackerID;
+                if (etrackerID) {
+                // Pageview parameters for overwrite
+                for (var i in etrackerConfig) {
+                    if (TagManager.utils.hasProperty(etrackerConfig, i)) {
+                        if(i.startsWith('et_')){
+                            window[i] = etrackerConfig[i];
+                        }
+                    }
+                }
+                // Pageview script
+                var s = document.getElementsByTagName('script')[0];

```

Comment:
or is inserting the script maybe automatically triggering a pageview? not quite sure how it works...

Reply:
If you insert the script it will trigger a pageview automatically.
As you might exclude some pages from having this code if have set it up as a pageview.

Your idea to fire it only once makes sense.

Reply:
Changed.